### PR TITLE
Adds DISA SCAP content

### PR DESCRIPTION
**Pull request by dalerichardson**

### CODE REVIEW

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 1]
```diff
@@ -166,26 +166,26 @@ Strong name assemblies that do not require verification in a development or test
...
 
 Application Configuration Files:
-Application configuration files contain settings specific to an application. If checking these files, a .NET review of a specific .NET application is most likely being conducted. These files contain configuration settings that the Common Language Runtime reads (such as assembly binding policy, remoting objects, and so on), and settings that the application can read.
+Application configuration files contain settings specific to an application. If checking these files, a .NET review of a specific .NET application is most likely being conducted. These files contain configuration settings that the Common Language Runtime reads (such as assembly binding policy, remoting objects, and so on), and settings that the application can read.
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:174]: Remove space at the end.
```suggestion
Application configuration files contain settings specific to an application. If checking these files, a .NET review of a specific .NET application is most likely being conducted. These files contain configuration settings that the Common Language Runtime reads (such as assembly binding policy, remoting objects, and so on), and settings that the application can read.
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 2]
```diff
@@ -166,26 +166,26 @@ Strong name assemblies that do not require verification in a development or test
           <xccdf:version update="http://iase.disa.mil/stigs">APPNET0062</xccdf:version>
           <xccdf:title>The .NET CLR must be configured to use FIPS approved encryption modules.</xccdf:title>
           <xccdf:description>&lt;VulnDiscussion&gt;FIPS encryption is configured via .NET configuration files.  There are numerous configuration files that affect different aspects of .Net behavior.  The .NET config files are described below.
-
+
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:169]: Remove space.
```suggestion

```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 3]
```diff
@@ -166,26 +166,26 @@ Strong name assemblies that do not require verification in a development or test
...
 
 Application Configuration Files:
-Application configuration files contain settings specific to an application. If checking these files, a .NET review of a specific .NET application is most likely being conducted. These files contain configuration settings that the Common Language Runtime reads (such as assembly binding policy, remoting objects, and so on), and settings that the application can read.
+Application configuration files contain settings specific to an application. If checking these files, a .NET review of a specific .NET application is most likely being conducted. These files contain configuration settings that the Common Language Runtime reads (such as assembly binding policy, remoting objects, and so on), and settings that the application can read. 
 
-The name and location of the application configuration file depends on the application's host, which can be one of the following:
+The name and location of the application configuration file depends on the application's host, which can be one of the following:
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:176]: Remove space at the end.
```suggestion
The name and location of the application configuration file depends on the application's host, which can be one of the following:
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 4]
```diff
@@ -166,26 +166,26 @@ Strong name assemblies that do not require verification in a development or test
...
 
 Application Configuration Files:
-Application configuration files contain settings specific to an application. If checking these files, a .NET review of a specific .NET application is most likely being conducted. These files contain configuration settings that the Common Language Runtime reads (such as assembly binding policy, remoting objects, and so on), and settings that the application can read.
+Application configuration files contain settings specific to an application. If checking these files, a .NET review of a specific .NET application is most likely being conducted. These files contain configuration settings that the Common Language Runtime reads (such as assembly binding policy, remoting objects, and so on), and settings that the application can read. 
 
-The name and location of the application configuration file depends on the application's host, which can be one of the following:
+The name and location of the application configuration file depends on the application's host, which can be one of the following: 
 
-Executable–hosted application configuration files.
+Executable–hosted application configuration files.
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:178]: Remove space at the end.
```suggestion
Executable–hosted application configuration files.
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 5]
```diff
@@ -166,26 +166,26 @@ Strong name assemblies that do not require verification in a development or test
...
 
-The name and location of the application configuration file depends on the application's host, which can be one of the following:
+The name and location of the application configuration file depends on the application's host, which can be one of the following: 
 
-Executable–hosted application configuration files.
+Executable–hosted application configuration files. 
 
-The configuration file for an application hosted by the executable host is in the same directory as the application. The name of the configuration file is the name of the application with a .config extension. For example, an application called myApp.exe can be associated with a configuration file called myApp.exe.config.
+The configuration file for an application hosted by the executable host is in the same directory as the application. The name of the configuration file is the name of the application with a .config extension. For example, an application called myApp.exe can be associated with a configuration file called myApp.exe.config.
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:180]: Remove space at the end.
```suggestion
The configuration file for an application hosted by the executable host is in the same directory as the application. The name of the configuration file is the name of the application with a .config extension. For example, an application called myApp.exe can be associated with a configuration file called myApp.exe.config.
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 6]
```diff
@@ -166,26 +166,26 @@ Strong name assemblies that do not require verification in a development or test
...
+The name and location of the application configuration file depends on the application's host, which can be one of the following: 
 
-Executable–hosted application configuration files.
+Executable–hosted application configuration files. 
 
-The configuration file for an application hosted by the executable host is in the same directory as the application. The name of the configuration file is the name of the application with a .config extension. For example, an application called myApp.exe can be associated with a configuration file called myApp.exe.config.
+The configuration file for an application hosted by the executable host is in the same directory as the application. The name of the configuration file is the name of the application with a .config extension. For example, an application called myApp.exe can be associated with a configuration file called myApp.exe.config. 
 
-Internet Explorer-hosted application configuration files.
+Internet Explorer-hosted application configuration files.
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:182]: Remove space at the end.
```suggestion
Internet Explorer-hosted application configuration files.
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 7]
```diff
@@ -166,26 +166,26 @@ Strong name assemblies that do not require verification in a development or test
...
 
-Internet Explorer-hosted application configuration files.
+Internet Explorer-hosted application configuration files. 
 
 If an application hosted in Internet Explorer has a configuration file, the location of this file is specified in a &lt;link&gt; tag with the following syntax.
 
 &lt;link rel="ConfigurationFileName" href="location"&gt;
 
-In this tag, "location" represents a URL that point to the configuration file. This sets the application base. The configuration file must be located on the same web site as the application.
+In this tag, "location" represents a URL that point to the configuration file. This sets the application base. The configuration file must be located on the same web site as the application.
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:188]: Remove space at the end.
```suggestion
In this tag, "location" represents a URL that point to the configuration file. This sets the application base. The configuration file must be located on the same web site as the application.
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 8]
```diff
@@ -201,8 +201,8 @@ In this tag, "location" represents a URL that point to the configuration file. T
           <xccdf:fixtext fixref="F-26917r468006_fix">Examine the .NET CLR configuration files to find the runtime element and then the "enforceFIPSPolicy" element.
 
 Example:
-&lt;configuration&gt;
-  &lt;runtime&gt;
+&lt;configuration&gt;
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:204]: Remove space at the end.
```suggestion
&lt;configuration&gt;
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 9]
```diff
@@ -201,8 +201,8 @@ In this tag, "location" represents a URL that point to the configuration file. T
           <xccdf:fixtext fixref="F-26917r468006_fix">Examine the .NET CLR configuration files to find the runtime element and then the "enforceFIPSPolicy" element.
 
 Example:
-&lt;configuration&gt;
-  &lt;runtime&gt;
+&lt;configuration&gt; 
+  &lt;runtime&gt;
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:205]: Remove space at the end.
```suggestion
  &lt;runtime&gt;
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 10]
```diff
@@ -220,7 +220,7 @@ Delete the "enforceFIPSPolicy" runtime element, change the setting to "true" or
         <xccdf:Rule id="xccdf_mil.disa.stig_rule_SV-225235r615940_rule" weight="10.0" severity="medium">
           <xccdf:version update="http://iase.disa.mil/stigs">APPNET0067</xccdf:version>
           <xccdf:title>Event tracing for Windows (ETW) for Common Language Runtime events must be enabled.</xccdf:title>
-          <xccdf:description>&lt;VulnDiscussion&gt;Event tracing captures information about applications utilizing the .NET CLR and the .NET CLR itself. This includes security oriented information, such as Strong Name and Authenticode verification.
+          <xccdf:description>&lt;VulnDiscussion&gt;Event tracing captures information about applications utilizing the .NET CLR and the .NET CLR itself. This includes security oriented information, such as Strong Name and Authenticode verification.
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:223]: Remove spaces at the end.
```suggestion
          <xccdf:description>&lt;VulnDiscussion&gt;Event tracing captures information about applications utilizing the .NET CLR and the .NET CLR itself. This includes security oriented information, such as Strong Name and Authenticode verification.
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 11]
```diff
@@ -235,7 +235,7 @@ Beginning with Windows Vista, ETW is enabled by default however, the .Net CLR an
           <xccdf:ident system="http://cyber.mil/cci">CCI-000130</xccdf:ident>
           <xccdf:fixtext fixref="F-26922r468021_fix">Open Windows explorer and search for all .NET config files including application config files (*.exe.config).
 
-Examine the configuration settings for
+Examine the configuration settings for
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:238]: Remove space at the end.
```suggestion
Examine the configuration settings for
```

File: scap/content/guides/disa/stig-dotnet4.xml [hunk 12]
```diff
@@ -267,7 +267,7 @@ Enable ETW Tracing by setting the etwEnable flag to "true" or obtain documented
 For 32-bit systems:
 HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\.NETFramework\v4.0.30319\
 
-For 64-bit systems:
+For 64-bit systems:
```
Comment (eemperor) [inline scap/content/guides/disa/stig-dotnet4.xml:270]: Remove space at the end.
```suggestion
For 64-bit systems:
```

File: scap/content/guides/openscap/build-content.log [hunk 13]
```diff
@@ -1,5 +1,4 @@
...
Updating files:  95% (10438/10987)
Updating files:  96% (10548/10987)
Updating files:  97% (10658/10987)
Updating files:  98% (10768/10987)
Updating files:  99% (10878/10987)
Updating files: 100% (10987/10987)
Updating files: 100% (10987/10987), done.
 /tmp/ComplianceAsCode/content /
 
 Tag to build: v0.1.70
```
Comment (eemperor) [inline scap/content/guides/openscap/build-content.log:4]: Looks like it's the same version.  Can you re-run the content update?  I just merged the PR that it's the version to 0.1.71.

Comment (dalerichardson) [reply scap/content/guides/openscap/build-content.log]: Updated branch and reran with new version.

**Review by eemperor — APPROVED** [review]

**Pull request merged**